doc/build_benchmark/run_benchmark.rst

```diff
@@ -0,0 +1,6 @@
+.. _run_benchmark:
+
+
+Run benchmark
```

Comment:
above we say `a Benchmark`: 
![image](https://github.com/benchopt/benchopt/assets/8993218/6f918f21-ba04-4230-b4e5-c63b88560d7f)